16.md

```diff
@@ -10,12 +10,15 @@ Relays may decide to allow replaceable and/or ephemeral events.
 
 Regular Events
 ------------------
-A *regular event* is defined as an event with a kind `1000 <= n < 10000`.
+A *regular event* is defined as an event with a kind in range `0 <= n < 10000` without events with kinds `0, 3, and 41` which are replaceable events.
 Upon a regular event being received, the relay SHOULD send it to all clients with a matching filter, and SHOULD store it. New events of the same kind do not affect previous events in any way.
 
+### Additional rules
+- Non-plaintext notes should use kind `1000 <= n < 10000`
+
```

Comment:
This is not a typo but the addition of a whole new rule?? Could we elaborate on this please? What is a non-plaintext note? What is a note?

Reply:
It was based on this row from NIP-01:
https://github.com/nostr-protocol/nips/blob/d41834fa5169d15ffd1519a773ee495c6076ecd3/01.md?plain=1#L101

Reply:
It was based on this row from NIP-01:
https://github.com/nostr-protocol/nips/blob/d41834fa5169d15ffd1519a773ee495c6076ecd3/01.md?plain=1#L101

Reply:
Just saying. The PR is not about typos but yeah, it's not a new rule. I had not known the rule from nip-01 and think it should be clearer. And I'm not sure if it should be maintained or dropped entirely as now kind ranges are about treatment, so where do I put my ephemeral binary message? Probably in the ephemeral range, not the binary range.